frontend/language/src/nb.json

```diff
@@ -218,25 +218,25 @@
   "deploy_to_test.deploy_helper_text_service_will_be_replaced": "Den tidligere versjonen vil bli erstattet når du legger ut appen på nytt",
   "deploy_to_test.deploy_in_progress": "Legger ut appen i testmiljøet, det vil ta ca. 1 minutt.",
   "deploy_to_test.error_a_problem_has_occured": "Det har oppstått et problem",
-  "deploy_to_test.error_service_was_not_deployed_check_title": "Noe gikk galt og appen ble ikke lagt ut i testmiljøet. Prøv å legge den ut på nytt.",
-  "deploy_to_test.error_service_was_not_deployed_title": "Appen ble ikke lagt ut i testmiljøet",
+  "deploy_to_test.error_service_was_not_deployed_check_title": "Noe gikk galt og appen ble ikke publisert i testmiljøet. Prøv å publisere den på nytt.",
+  "deploy_to_test.error_service_was_not_deployed_title": "Appen ble ikke publisert i testmiljøet",
   "deploy_to_test.error_there_is_something_wrong_with_your_environment_part1": "Det er noe galt med ditt",
   "deploy_to_test.error_there_is_something_wrong_with_your_environment_part2": "-miljø. Vennligst kontakt support.",
```

Comment:
Linjene part 1 part 2 eksisterer bare i tekstfilene og kan slettes, ifølge Lars W

Reply:
Det ser ut for meg som dette stemmer for alle nøklene som starter med `deploy_to_test` 👍

Reply:
Usikker på om disse linjen ble tatt bort av Michael? jeg husker dessverre ikke.